packages/web-app-epub-reader/src/App.vue

```diff
@@ -163,9 +237,17 @@ export default defineComponent({
       navigateRightDisabled,
       currentChapter,
       chapters,
-      showChapter
+      showChapter,
+      resetFontSize,
+      increaseFontSize,
+      decreaseFontSize,
+      increaseFontSizeDisabled,
+      decreaseFontSizeDisabled,
+      fontSizePercentage,
+      FONT_SIZE_PERCENTAGE_STEP
     }
-  }
+  },
+  methods: { $gettext }
```

Comment:
No options API please. Why do you need it at all? When I remove it my IDE still shows $gettext correctly in the template...

Reply:
I don't need it at all, I marked a comment already. My ide is always auto importing it 🤬

Reply:
ah, ok, so only a reminder for yourself 👍